content/engineering/licenses.md

```diff
@@ -0,0 +1,32 @@
+# Licenses
+
+This page documents which licenses are usable at Sourcegraph. For questions, please contact #legal in Slack.
+
+## Licensing Sourcegraph code
+
+All Sourcegraph code should generally be licensed under either:
+
+- The [Apache license](https://github.com/sourcegraph/sourcegraph/blob/main/LICENSE.apache)
+- The [Sourcegraph enterprise license](https://github.com/sourcegraph/sourcegraph/blob/main/LICENSE.enterprise)
+
+Which one to choose is a decision that your team, manager, and product/leadership should agree on. In general core value adds should be enterprise-licensed, while utilities, support libraries, etc. that are unlikely to benefit potential competitors should be Apache licensed.
+
+## Using open source code
+
+### Allowed licenses: Permissive Licenses
+
+You may use any code licensed under any permissive license, such as Apache, MIT, BSD. If you aren’t sure whether a license is permissive, see the [permissive license list](https://blueoakcouncil.org/list) compiled by Blue Oak Council.
+
+### Not allowed licenses: Copyleft Licenses
+
+We avoid using code and libraries under copyleft licenses. If you can't find an alternative, reach out to the legal team. Below are the most common copyleft licenses:
+
+- AGPL
+- GPL (ok for internal use)
+- LGPL (ok for internal use)
+- CCDL (ok for internal use)
+- Eclipse Public License (ok for internal use)
+- Mozilla Public License (ok for internal use)
+- Anything else not on the permissive license list
+
+If it's for internal use only and not for distribution to users/customers, then tell the legal team and the licenses above will likely be approved.
```

Comment:
@chrismwendt  Note that this is semi-enforced here: https://github.com/sourcegraph/sourcegraph/blob/main/doc/dependency_decisions.yml we should make sure these lists stay in sync

Reply:
https://docs.sourcegraph.com/dev/background-information/continuous_integration#third-party-licenses